app/views/layouts/_cms.html.erb

```diff
@@ -1,15 +1,9 @@
 <%= javascript_include_tag 'vendor/jquery/extensions/jquery.cookie.js' %>
-<%= javascript_include_tag 'jquery.pjax.js' %>
 <%= javascript_include_tag 'provider/admin/cms/sidebar' %>
 
 <%= javascript_packs_with_chunks_tag 'codemirror', 'cms' %>
 <%= javascript_include_tag 'codemirror/modes/liquid' %>
 
-<%= javascript_tag do %>
-  <% timeout = (Rails.env.development? or Rails.env.test?) ? 10000 : 3000 -%>
-  $.extend($.pjax.defaults, { timeout: <%= timeout %> });
```

Comment:
I think this was added as a hack/workaround very long time ago, and we shouldn't need it anymore. Default timeout is 650.